ci/parse-trivy-results.py

```diff
@@ -0,0 +1,155 @@
+flags.DEFINE_string(
+    "release_tag", None, "Number of the latest release/tag. "
+    "This can be retrieved using \n "
+    "git describe --abbrev=0 --tags > _release_tag")
+flags.DEFINE_string(
+    "repo_name", None,
+    "Name of the github repo where potential issues will be created.")
+flags.DEFINE_string(
+    "token_file", None,
+    "Name of the file where the GitHub token has been stored.\n"
+    "This can be created using \n"
+    "gcloud secrets versions access latest "
+    "--secret=Ts-bridge-bot-token --format='get(payload.data)' | "
+    "tr '_-' '/+' | base64 -d > git_token.txt")
+flags.DEFINE_string("trivy_file", None,
+                    "Name of the files where results from Trivy are stored")
+
+flags.mark_flag_as_required("build_id")
+flags.mark_flag_as_required("commit_id")
+flags.mark_flag_as_required("release_tag")
+flags.mark_flag_as_required("repo_name")
+flags.mark_flag_as_required("token_file")
+flags.mark_flag_as_required("trivy_file")
+
+
+def validate_flags():
+    error = None
+    if not Path("{}.json".format(FLAGS.trivy_file)).is_file():
+        error = ("Please run \n"
+                 "trivy image --format json --light --no-progress -o {}.json "
+                 "gcr.io/cre-tools/ts-bridge \n").format(FLAGS.trivy_file)
+    elif not Path("{}.table".format(FLAGS.trivy_file)).is_file():
+        error = ("Please run \n"
+                 "trivy image --format --light --no-progress -o {}.table "
+                 "gcr.io/cre-tools/ts-bridge \n").format(FLAGS.trivy_file)
+    elif not Path(FLAGS.token_file).is_file():
+        error = ("Please run \n"
+                 "gcloud secrets versions access latest "
+                 "--secret=Ts-bridge-bot-token --format='get(payload.data)' | "
+                 "tr '_-' '/+' | base64 -d > {}").format(FLAGS.token_file)
+    if error:
+        sys.exit(error)
+
+
+def load_results():
+    """ Load the results from Trivy."""
+    trivy_out_json = "{}.json".format(FLAGS.trivy_file)
+    trivy_out_table = "{}.table".format(FLAGS.trivy_file)
+    with open(trivy_out_json) as f:
+        # trivy_result is the json result used for parsing vulnerabilities
+        # Index is 0 because there is only one target built.
+        trivy_result = json.load(f)[0]
+    with open(trivy_out_table) as f:
+        # trivy_table is the results in a more readable table format which can
+        # be included in GitHub issues.
+        trivy_table = f.read()
+    return [trivy_result, trivy_table]
+
+
+def get_severity_list(vulnerabilities):
+    """Filters out all the severities in a given list of vulnerabilities."""
+    severity_list = [v.get("Severity") for v in vulnerabilities]
+    return sorted(set(severity_list))
+
+
+def get_github_repo():
+    """Connects to GitHub API and returns a repo object."""
+    with open(FLAGS.token_file) as f:
+        token = f.read()
+    github = Github(token)
+    return github.get_repo(FLAGS.repo_name)
```

Comment:
We should fail gracefully if `get_repo` fails.

Reply:
Added exception handling to address this